sympy/strategies/tests/test_traverse.py

```diff
@@ -1,65 +1,80 @@
-from sympy.strategies.traverse import (top_down, bottom_up, sall, top_down_once,
-        bottom_up_once, basic_fns)
+from sympy.strategies.traverse import (
+    top_down,
+    bottom_up,
+    sall,
+    top_down_once,
+    bottom_up_once,
+    basic_fns
+)
```

Comment:
The previous version is how the majority of the SymPy codebase deals with this kind of import. In general, you should follow the conventions that already exist in the codebase.